README.md

```diff
@@ -133,6 +140,25 @@ repository’s Security tab. For more information, see the Scorecards GitHub
 Action
 [installation instructions](https://github.com/ossf/scorecard-action#installation).
 
+### Scorecards REST API
+
+To query pre-calculated scores of OSS projects use the REST API
+[here](https://api.securityscorecards.dev). To enable your project to be
```

Comment:
Is this URL`https://api.securityscorecards.dev` going to change? Do we want to wait for any name change?

Reply:
I hope it'll change, but I haven't heard back from operations@openssf for quite a while now. I don't want to delay the launch for a domain name, so in case it changes I'll update it, else would probably launch with this domain.

Reply:
👍